include/alpaka/meta/ApplyTuple.hpp

```diff
@@ -12,8 +12,6 @@
 #include <alpaka/core/Common.hpp>
 #include <alpaka/core/Unused.hpp>
 
-#include <alpaka/meta/IntegerSequence.hpp>
```

Comment:
Can't this file itself be removed then as well? :)

Reply:
It will get removed in the future but there are other usages that have to be replaced as well. I am already working on it but there seem to be some compiler issues.